src/styles/bramble_overrides.css

```diff
@@ -72,6 +72,29 @@
     background: rgba(0, 135, 207, .3) !important;
 }
 
+table.guide {
+	width: 100%;
```

Comment:
Your indents are all too far in this file.  4-space indent please, match to surrounding code.